opencti-platform/opencti-front/lang/front/en.json

```
@@ -996,6 +999,7 @@
   "Do you want to delete this decay rule?": "Do you want to delete this decay rule?",
   "Do you want to delete this dissemination list?": "Do you want to delete this dissemination list?",
   "Do you want to delete this draft?": "Do you want to delete this draft?",
+  "Do you want to delete this Email template?": "Do you want to delete this Email template?",
```

```suggestion
  "Do you want to delete this Email template?": "Do you want to delete this email template?",
```
Any reason for a capital 'E' here ?